docs/known_bugs.rst

```diff
@@ -84,14 +84,24 @@ where you do not see this error or use Jupyter Notebook instead.
 Cubeviz
 -------
 
+Cubeviz sometimes fails to run from command line interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Running Cubeviz from the command line sometimes results in a failure
+to initialize the app in the browser due to a ``RuntimeError`` in
+``tornado/ioloop.py``. We are investigating, but in the meantime
+reinstalling fresh in a new conda environment may help. Alternatively,
```

Comment:
I thought you didn't see the problem until you installed a fresh env? 🤔 

Regardless, a known issue is the right track for now. Thanks!